### PR TITLE
 Kube postdeployment

### PR DESCRIPTION
**Pull request by devtrap**
- add kubernetes post deployment 
- support deployment, pod, networkPolicy,podSecurityPolicy,service,serviceaccount,replicaset and rolebinding
- add different kubernetes config cores 
- add trace back for main exception 
- add db support for kubernetes post deployment

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -4,21 +4,22 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (farchide) [inline .vscode/launch.json]: @devtrap 
remove this file from the commit

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "testenv/bin/python"
+    "python.pythonPath": ".venv/test/bin/python"
```
Comment (farchide) [inline .vscode/settings.json]: @devtrap 
remove this file from commit

**Review by farchide — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (devtrap) [reply .vscode/settings.json]: reverted

File: .vscode/launch.json (hunk 1 above)
Comment (devtrap) [reply .vscode/launch.json]: removed in updated pull request

**Review by vatsalgit5118 — APPROVED** [review]

**Review by farchide — APPROVED** [review]

**Pull request merged**